### PR TITLE
Removing the BTC faucet functionality because BlueWallet does not support the Bitcoin testnet 

### PR DESCRIPTION
**Pull request by 3yekn**
We need to remove the BTC testnet faucet because of the signer configuration. We provided BTC secrets to the evaluator for use in testing.

### CODE REVIEW

**Review by Noc2 — APPROVED** [review]
Thanks for the update. Sounds good to me. For reference: https://github.com/w3f/Grant-Milestone-Delivery/pull/605

**Review by takahser — APPROVED** [review]

**Comment by alxs** [comment]
Application PR: https://github.com/w3f/Grants-Program/pull/868

**Review by alxs — APPROVED** [review]

File: applications/native-bitcoin-vaults.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ The following Extrinsics or RPC calls, with relevant UI functions, will be inclu
 | 1. | PSBT Signing | Users will be able to sign transactions using our customized signer (`Spectre Desktop`) and save the output to the Substrate node. |  
 | 2. | Transaction Broadcast | When the threshold of signatures has been reached, the node will broadcast the transaction to the Bitcoin network (via integrated Bitcoin core node) |  
 | 4. | Hosted Instances | We will host a `testnet` and `mainnet` instance of the full solution (as a standalone Substrate chain) for no less than 2 years |  
-| 5. | EZ-Try | The hosted instances will have `EZ-Try`, which is our method of ensuring any level user can easily go to the app with no prerequisites and try it. (e.g. auto-faucet, auto-generate `xpubs`) |  
+
```
Comment (alxs) [inline applications/native-bitcoin-vaults.md:293]:
```suggestion
```

**Review by alxs — CHANGES_REQUESTED** [review]
This breaks the formatting

**Review by alxs — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Review by semuelle — APPROVED** [review]

**Review by takahser — APPROVED** [review]

**Review by keeganquigley — APPROVED** [review]

**Pull request merged**